install/kubernetes/cilium/values.yaml.tmpl

```
@@ -2571,3 +2571,16 @@ dnsProxy:
 sctp:
   # -- Enable SCTP support. NOTE: Currently, SCTP support does not support rewriting ports or multihoming.
   enabled: false
+
+auth:
+  mTLS:
+    # -- Enable mtls-spiffe authentication method in CiliumNetworkPolicy
+    enabled: false
+    # -- SPIRE socket path where the SPIRE agent is listening
+    spireSocketPath: /run/spire/sockets/
+    # -- SPIRE admin socket file name to use for fetching certificates
+    spireAdminSocket: admin.sock
```

Can we simplify and use one value for the full socket path here? e.g.
```suggestion
    # -- SPIRE socket path where the SPIRE agent is listening
    spireSocketPath: /run/spire/sockets/admin.sock
```
When the directory is needed from the helm template, we can then use `{{ dir .Values.auth.mTLS.spireSocketPath }}`. See also the [Hubble socket Helm value](https://github.com/cilium/cilium/blob/1e09c396bea97af265fe70d4e7f8f09b5ae01c07/install/kubernetes/cilium/values.yaml#L924-L925) and [Hubble Relay's  `volumeMounts` setup](https://github.com/cilium/cilium/blob/1e09c396bea97af265fe70d4e7f8f09b5ae01c07/install/kubernetes/cilium/templates/hubble-relay/deployment.yaml#L85-L89).

---

TIL